gulpfile.js

```diff
@@ -81,6 +83,22 @@ var DEFINES = {
   PDFJS_NEXT: false,
 };
 
+function safeSpawnSync(command, parameters, options) {
+  // Windows has problems executing commands outside a shell.
+  if (process.platform === 'win32') {
+    options = options || {};
+    options.shell = true;
+  }
+
+  var result = spawnSync(command, parameters, options);
+  if (result.status) {
```

Comment:
You're absolutely right, now that I look at the code again! I have changed it in the new commits.